geometry_msgs/PoseArray poses
---
string success
